generateIOS.rb

```diff
@@ -31,42 +31,43 @@
   is_bridging_header_created = false
   project.targets.each do |target|
     # Skip if target type is unit tests or app extension, etc.
-    next if target.product_type != "com.apple.product-type.application"
-
-    target.build_configurations.each do |config|
-      if config.build_settings['SWIFT_OBJC_BRIDGING_HEADER'] == nil
-        # Create default bridging header if needed and link it to build settings
-        if File.exist?(default_bridging_header_path) == false
-          bridging_header_file = File.new(default_bridging_header_path, 'w+')
-          File.write(default_bridging_header_path, bridging_headers, mode: 'a')
-          bridging_header_file.close
-          is_bridging_header_created = true
-          puts "Created #{default_bridging_header_filename}"
+    if target.product_type == ("com.apple.product-type.application" || "com.apple.product-type.library.static")
```

Comment:
This looks wrong?
```ruby
$ irb
irb(main):001:0> "com.apple.product-type.application" || "com.apple.product-type
.library.static"
=> "com.apple.product-type.application"
```
So unless I'm missing something, that line seems to be equivalent to:
```ruby
    if target.product_type == "com.apple.product-type.application"
```
I think what you want instead is:
```ruby
if ["com.apple.product-type.application", "com.apple.product-type.library.static"].include? target.product_type
```

Testing:
```ruby
$ irb
irb(main):001:0> product_type = "com.apple.product-type.application"
=> "com.apple.product-type.application"
irb(main):002:0> ["com.apple.product-type.application", "com.apple.product-type.library.static"].include? product_type
=> true
irb(main):003:0> product_type = "com.apple.product-type.library.static"
=> "com.apple.product-type.library.static"
irb(main):004:0> ["com.apple.product-type.application", "com.apple.product-type.library.static"].include? product_type
=> true
irb(main):005:0> product_type = "test"
=> "test"
irb(main):006:0> ["com.apple.product-type.application", "com.apple.product-type.library.static"].include? product_type
=> false
irb(main):007:0> 
```

Reply:
I'm not quite sure what you're asking :)

The following:
```ruby
    if target.product_type == ("com.apple.product-type.application" || "com.apple.product-type.library.static")
```
is equivalent to:
```ruby
    if target.product_type == "com.apple.product-type.application"
```
because `"string1" || "string2"` evaluates to `"string1"`.

The following:
```ruby
if ["com.apple.product-type.application", "com.apple.product-type.library.static"].include? target.product_type
```
means: "If target.product_type is equal to either of those two strings, then..."
i.e. "Does this array include (contain) target.product_type?"

Reply:
@mateusz1913 I assume the intention of that line is to say 'If target.product_type is "com.apple.product-type.application" or "com.apple.product-type.library.static" then do the following'.

What it is actually doing is 'If target.product_type is "com.apple.product-type.application" then do the following'.

To get the first meaning you should do:
```ruby
if list_of_possible_values.include?(target.product_type)
```

This is like the following Python code:
```python
if target.product_type in list_of_possible_values
```